Config/routes.js

```diff
@@ -2,7 +2,7 @@ const ToppingsController = require("../Controllers/ToppingsController");
 
 const routes = (app) => {
   app.get("/", ToppingsController.getAllToppings);
```

Comment:
Suggest using a better route name than just nothing, it would be quite unusual to have an API endpoint that is just /.

suggest /toppings or something? Same with the put route

Reply:
The reason is because the combo of the HTTP verb and the route name should pretty much describe what the route is doing - so a PUT to /toppings makes it pretty obvious that you will be updating toppings, whereas a PUT to / is just meaningless